src/cameras/PerspectiveCamera.js

```diff
@@ -135,6 +151,18 @@ THREE.PerspectiveCamera.prototype.copy = function ( source ) {
 	this.near = source.near;
 	this.far = source.far;
 
+	this.zoom = source.zoom;
+	this.focalLength = source.focalLength;
+
+	this.skewSlopeX = source.skewSlopeX;
+	this.skewSlopeY = source.skewSlopeY;
+
+	if ( source.view !== null && source.view !== undefined ) {
```

Comment:
>> I think only checking for `null` should be enough?

> It's called by the loader and since it didn't get serialized before might be missing, thus `=== undefined`.

Which loader?